src/main/java/org/prebid/server/proto/openrtb/ext/ExtPrebid.java

```diff
@@ -13,6 +16,8 @@
 @Value
 public class ExtPrebid<P, B> {
 
+    public static final String AUCTION_ENVIRONMENT_KEY = "ae";
```

Comment:
I don't think that we need this constant.

Reply:
Removed it. Was meant to indicate the `ae` in `validateFledgeConfig()` is the same as this one.